committeeoversightapp/templates/partials/hearing_table.html

```diff
@@ -2,9 +2,10 @@
   <table class="table table-striped table-borderless {{ table_size }}" id="hearing-table">
      <thead>
        <tr>
-         <th scope="col">Date Updated</th>
-         <th scope="col" class="hearing-title-col">Hearing Title</th>
-         <th scope="col">Hearing Date</th>
+         <th scope="col" style="min-width:10%;">Date</th>
```

Comment:
TIL you should do this, instead of the `col-*` classes, in Bootstrap 4! Thank you!!!